kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/timeout/CoroutinesTimeoutOverrideTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.sksamuel.kotest.engine.test.timeout
+
+import io.kotest.core.spec.style.FunSpec
+import kotlin.time.Duration.Companion.milliseconds
+
+class CoroutinesTimeoutOverrideTest : FunSpec({
```

Comment:
```suggestion
@DoNotParallelize
class CoroutinesTimeoutOverrideTest : FunSpec({
```

`System.setProperty` affects global state.